src/utils/Cache.js

```diff
@@ -39,3 +39,8 @@ module.exports.guildCache = new Map();
  * @type {Map<string, string>}
  */
 module.exports.afkUsers = new Map();
+
+/**
+ * @type {Map<string, string>}
+ */
+module.exports.video = null
```

Comment:
Video is null, can we setup a proper defination what is it supposed to be?

Reply:
I made it so that it checks for a new video every 3 minutes, and if .video is null or .video is similar to the newest checked video's Id, it returns and won't execute the rest of the code

Reply:
@HmmNoPls Then can you please set the @type definination to be whatever it is suposed to be? Like a integer? a array? what is it supposed to be